sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -715,14 +715,14 @@ def std_overload(df, axis=None, skipna=None, level=None, ddof=1, numeric_only=No
 
     Limitations
     -----------
-    Parameters axis, skipna, level, ddof, numeric_only unsupported.
+    Parameters axis, skipna, level, ddof, numeric_only are unsupported.
```

Comment:
Please, highlight parameter names and use **and** between the last two:
```suggestion
    Parameters ``axis``, skipna, level, ddof and numeric_only are unsupported.
```